### PR TITLE
test(rbac): add playwright tests for the plugin

### PR DESCRIPTION
**Pull request by invincibleJai**
playwright tests for the RBAC plugin in dev setup

Tracks: https://issues.redhat.com/browse/RHIDP-1463

### CODE REVIEW

File: plugins/rbac/tests/rbac.spec.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    const policies =
+      /^(?:(Read|Create|Update|Delete)(?:, (?:Read|Create|Update|Delete))*|Use)$/;
+    await verifyCellsInTable([policies], page);
+
+    page.locator(`a`).filter({ hasText: 'RBAC' }).click();
+  });
+
+  test('Edit user from overview page', async () => {
+    const roleName = 'role:default/rbac_admin';
+    page.locator(`a`).filter({ hasText: roleName }).click();
```
Comment (jrichter1) [inline plugins/rbac/tests/rbac.spec.ts:94]: missing await

File: plugins/rbac/tests/rbac.spec.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      timeout: 20000,
+    });
+    overviewTabSelector.click();
+
+    const RoleOverviewPO = {
+      updatePolicies: 'span[data-testid="update-policies"]',
+      updateMembers: 'span[data-testid="update-members"]',
+    };
+
+    await page.click(RoleOverviewPO.updateMembers);
```
Comment (jrichter1) [inline plugins/rbac/tests/rbac.spec.ts:109]: recommend using locator click method instead

File: plugins/rbac/tests/rbac.spec.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+test.describe('RBAC plugin', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('/');
+    const navSelector = 'nav [aria-label="Administration"]';
+    page.click(navSelector);
```
[the remark this answers is not in the document]
Comment (invincibleJai) [reply plugins/rbac/tests/rbac.spec.ts]: Thanks @jrichter1 updated , https://playwright.dev/docs/api/class-page#page-click

File: plugins/rbac/tests/rbac.spec.ts [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+test.describe('RBAC plugin', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('/');
+    const navSelector = 'nav [aria-label="Administration"]';
+    page.locator(navSelector).click();
```
Comment (jrichter1) [inline plugins/rbac/tests/rbac.spec.ts:18]: clicks are still async, `await` them :)

Comment (invincibleJai) [reply plugins/rbac/tests/rbac.spec.ts]: Sorry, missed this. Have updated it

**Review by gashcrumb — APPROVED** [review]
:+1:

**Review by jrichter1 — APPROVED** [review]

**Pull request merged**